create_machine.sh

```diff
@@ -41,4 +41,4 @@ else
 fi
 ./user_data.sh "${MACHINE_NAME}" "${OS_TYPE}" | kubectl apply -n metal3 -f -
 
-make_machine | kubectl apply -n metal3 -f -
+make_machine | kubectl apply -n metal3 -f -
```

Comment:
You have made a CR file (machine.yaml) for this (please place it in v1alpha1 folder), you can use it in the make_machine function